### ISSUE TITLE
Randomly failed in ubuntu vm of GitHub Actions 

### ISSUE THREAD

**Issue by hsbt** [bundler/bundler#7336]
I test bundler examples every commits in https://github.com/ruby/ruby.

I got test failure only ubuntu VM randomly.

```
NameError: uninitialized constant Bundler::Molinillo::Resolver::Resolution::PP
        /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:610:in `rescue in block in create_conflict'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:607:in `block in create_conflict'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:606:in `each'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:606:in `create_conflict'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:256:in `process_topmost_state'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolution.rb:182:in `resolve'
         /home/runner/work/ruby/ruby/lib/bundler/vendor/molinillo/lib/molinillo/resolver.rb:43:in `resolve'
         /home/runner/work/ruby/ruby/lib/bundler/resolver.rb:50:in `start'
         /home/runner/work/ruby/ruby/lib/bundler/resolver.rb:22:in `resolve'
(snip)

```

https://github.com/ruby/ruby/runs/207405029#step:10:82

I'm not sure why it's happened. Does anyone have the idea related to this?

**Comment by nobu** [comment]
Probably fixed by ruby/ruby@b870ca58791e576f5dbb263f54ae433a7f6d65ee.

**Comment by hsbt** [comment]
Fixed at https://github.com/bundler/bundler/pull/7344
